tasks/theme.js

```diff
@@ -20,6 +20,10 @@ module.exports = function(grunt) {
    *       "sourcemap": true
    *     }
    *   },
+   *   "legionaire": {
+   *     "path": "<%= config.srcPaths.drupal %>/themes/legionaire",
+   *     "proxy": true
```

Comment:
For the task automatically attached to compile-theme? That makes sense. How about proxy property can instead be an object, with a default property that will override "compile"?

I hesitate to default to 'default' because that is often a watch task and task-not-found seems like a more obvious error state.